packages/block-editor/src/components/use-block-display-information/README.md

```diff
@@ -1,11 +1,16 @@
 # useBlockDisplayInformation
 
-A React Hook that tries to find a matching block variation and returns the appropriate information for display reasons. In order to try to find a match we need two things:
+Hook used appropriate information for display reasons.
```

Comment:
> Hook used appropriate information for display reasons.

Not sure how best to revise this. Did you mean "Hook used to return appropriate information for display reasons" ?

Reply:
Duh, looks like I'm unusable after 6 PM. Fixed them, thanks!